esphome/components/esp32_ble_tracker/esp32_ble_tracker.h

```diff
@@ -121,6 +123,7 @@ class ESPBTDevice {
   std::vector<ESPBTUUID> service_uuids_;
   std::vector<ServiceData> manufacturer_datas_{};
   std::vector<ServiceData> service_datas_{};
+  esp_ble_gap_cb_param_t::ble_scan_result_evt_param scan_rst_{};
```

Comment:
`rst` is confusing, lets just name it `result`
```suggestion
  esp_ble_gap_cb_param_t::ble_scan_result_evt_param scan_result_{};
```

Reply:
Fixed.
PS: Previously I did it to fit existing parse_scan_rst.

Reply:
Ah ok. I will change that too. Thanks